apps/website-25/src/lib/utils.ts

```diff
@@ -0,0 +1,3 @@
+export const splitCommaString = (input: string | null | undefined): string[] => {
+  return input ? input.split(',').filter((item) => item !== '') : [];
+};
```

Comment:
we could abstract this to something more like:
```suggestion
export const formatStringToArray = (input: string | null | undefined, splitBy: string): string[] => {
  return input ? input.split(splitBy).map((o) => o.trim()).filter((item) => item !== '') : [];
};
```
that way we can use it across more general airtable formatting usecases:
```
// users/me.ts
const courseNames = formatStringToArray(user?.courseSitesVisited, ',');

// MultipleChoice.tsx
const formattedOptions = formatStringToArray(options, '/n');
```

i think trim() will likely be used in all cases. if in the future we find one that doesn't require it, we could always add a new option doNotTrim or something to handle it

i find having this helper useful since we know airtable doesn't handle arrays, so it is likely we will continue encountering it.

Reply:
Seems reasonable 👍. I've updated it to this now and applied it the the one case there is at the moment